tests/TestCase/Shell/CompletionShellTest.php

```diff
@@ -258,7 +258,7 @@ public function testSubCommandsAppDuplicatePluginNoDot()
         $this->Shell->runCommand(['subcommands', 'sample']);
         $output = $this->out->output;
 
-        $expected = "derp load sample\n";
+        $expected = "derp load returnValue sample withAbort\n";
```

Comment:
You might want to exchange this with some `$this->assertContains()` assertions instead, to prevent future breaks.

(If not, no big deal, it'll be changed in the future cli integration test PR as I ran into the same issue.)